client/src/routes/admin/challs.js

```diff
@@ -14,16 +14,19 @@ const SAMPLE_PROBLEM = {
   author: '',
   files: [],
   points: {
-    min: 0,
-    max: 0
+    min: 100,
+    max: 500
   }
 }
 
 const Challenges = ({ classes }) => {
   const [problems, setProblems] = useState([])
-  const [newId, setNewId] = useState(uuid())
+  const newId = useMemo(() => uuid(), [problems])
```

Comment:
```suggestion
  // eslint-disable-next-line react-hooks/exhaustive-deps
  const newId = useMemo(() => uuid(), [problems])
```
Potentially add an explanatory comment as well